test/src/test/resources/transport/nettyConfigDynamicStreams.properties

```diff
@@ -0,0 +1,16 @@
+# Transport Plugin Configuration
```

Comment:
no need to have a different properties file.  We are just changing the implementation in an existing plugin